soc/arm64/xenvm/mmu_regions.c

```diff
@@ -18,6 +18,11 @@ static const struct arm_mmu_region mmu_regions[] = {
 			      DT_REG_ADDR_BY_IDX(DT_INST(0, arm_gic), 1),
 			      DT_REG_SIZE_BY_IDX(DT_INST(0, arm_gic), 1),
 			      MT_DEVICE_nGnRnE | MT_P_RW_U_RW | MT_NS),
+
+	MMU_REGION_FLAT_ENTRY("HYPERVISOR",
```

Comment:
@carlocaione I remember you are working on this? And @luca-fancellu has a similar PR https://github.com/zephyrproject-rtos/zephyr/pull/46300/files#diff-672e451c592cc783c11079c204f12dc4317823ad70ae21be6e752d5699ca6055? Do you have any comments?

Reply:
I'm doing a write up of the current MMU mapping situation just right now. For now let's allow having this kind of mapping.